pkg/service/protocolDataService/protocol.go

```diff
@@ -555,7 +561,7 @@ func (pds *ProtocolDataService) ListWithdrawalsForStrategies(ctx context.Context
 			) AS expanded
 			WHERE event_name = 'WithdrawalQueued'
 				AND strategy IN @strategies
-				AND output_data->>'withdrawalRoot' NOT IN (SELECT withdrawal_root FROM completed_roots)
+				AND output_data->>'withdrawalRoot' NOT IN (SELECT withdrawal_root FROM completed_regular_roots)
```

Comment:
Regular withdrawal queued events may be completed as slashing roots. This doesn't work, I think.